doc/releases/changelog-0.37.0.md

```diff
@@ -6,17 +6,79 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations using the
+  [`quimb` backend](https://quimb.readthedocs.io/en/latest/).
```

Comment:
This rendered kind of strangely:
![image](https://github.com/PennyLaneAI/pennylane/assets/67476785/c1bfeb13-70d7-4bd6-b462-332138775536)

Reply:
Thanks, I forgot that - this shows fine in markdown but does not convert correctly to the docs version.